format_spec.md

```diff
@@ -0,0 +1,339 @@
+# MCIP Format Specification
+
+This document provides detailed information on the MCIP Format.
+
+The MCIP format uses JSON to store information. If providing an API that returns format-compliant information, the `application/json` Content-Type header SHOULD be provided. If stored on disk, the `.json` extension SHOULD be used.
+
+Metadata MAY be modularized in order to save bandwidth or storage space. Information on what fields may be modularized and the proper method of doing it will be at the end of this document.
+
+An format-compliant example is provided in `format_example.json`
+
+## Fields
+
+Fields are stored inside the JSON metadata of a Project. Fields that are optional will be marked as such. All fields are stored as a String value unless stated otherwise. Below is a complete list of all fields available for the MCIP Format, grouped into categories.
+
+## General Project Information
+
+### `schemaVersion`
+
+This field MUST be stored as an integer. It contains the version of the Format being used, to help with backwards compatibility.
+
+---
+
+### `id`
+
+The ID of the Project. There is no standard or required format on how to store an ID, other than it CANNOT include spaces. It may be a slug, UUID, random number, or other method of storing IDs. It is RECOMMENDED that IDs are not tied to the name of a project.
+
+---
+
+### `name`
+
+The name of a Project. This name MUST be a human readable name and SHOULD not contain unnecessary information. For instance, the name field should not specify where the project is being hosted, or the authors of a project.
+
+---
+
+### `summary` (optional)
+
+A short, one sentence summary of the Project. Any form of rich text such as HTML SHOULD NOT be used.
```

Comment:
If I get a file with HTML in this field, should I display it as text, display it as HTML, or remove the HTML?